File: src/main/old/bbarena/model/event/BaseEventFlowListener.java
package bbarena.model.event;

/**
 * Implementations of this interface will be notified of EventLine changes
 *
 * @author f.bellentani
 */
public abstract class BaseEventFlowListener implements EventFlowListener {

    private EventManager _eventManager;

    @Override
    public void setEventManager(EventManager eventManager) {
        _eventManager = eventManager;
    }

    public EventManager getEventManager() {
        return _eventManager;
    }
}

File: src/main/kotlin/bbarena/model/event/BaseEventFlowListener.kt
package bbarena.model.event

/**
 * Implementations of this interface will be notified of EventLine changes
 *
 * @author f.bellentani
 */
abstract class BaseEventFlowListener : EventFlowListener {

    private var _eventManager: EventManager? = null

    override fun setEventManager(eventManager: EventManager) {
        _eventManager = eventManager
    }

    fun getEventManager(): EventManager? {
        return _eventManager
    }
}
